datasync/chaindatafetcher/kas/model.go

```diff
@@ -21,3 +24,15 @@ type Tx struct {
 func (Tx) TableName() string {
 	return TxTableName
 }
+
+type RevertedTx struct {
+	TransactionHash []byte `gorm:"column:transactionHash;type:VARBINARY(32);NOT NULL;PRIMARY_KEY"`
+	BlockNumber     int64  `gorm:"column:blockNumber;type:BIGINT"`
+	RevertMessage   string `gorm:"column:revertMessage;type:VARCHAR(1024)"`
+	ContractAddress []byte `gorm:"column:contractAddress;type:VARBINARY(20);NOT NULL"`
+	Timestamp       int64  `gorm:"column:timestamp;type:INT(11)"`
+}
+
+func (RevertedTx) TableName() string {
```

Comment:
```suggestion
func (revertedTx *RevertedTx) TableName() string {
```

Reply:
I think this method is used under the interface only on `gorm` package.. So, receiver is not really necessary.